src/components/account/UserProfile.tsx

```diff
@@ -69,7 +69,7 @@ class UserProfile extends React.Component<Props> {
         variant="contained"
         href={`https://github.com/marketplace/cirrus-ci/order/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW45OTI=?account=${user.githubUserName}`}
       >
-        <Icon className={classNames(classes.leftIcon, 'fa', 'fa-github')} />
+        <GitHubIcon className={classes.leftIcon} />
```

Comment:
You should use the `startIcon` prop if you are adding icons to buttons for enhanced flexibility.